frontend/src/concepts/connectionTypes/fields/ConnectionTypeFormFields.tsx

```diff
@@ -47,6 +48,22 @@ const ConnectionTypeFormFields: React.FC<Props> = ({
     [fields],
   );
 
+  const unmatchedValues: ConnectionTypeDataField[] = React.useMemo(() => {
+    const unmatched: ConnectionTypeDataField[] = [];
+    for (const key in connectionValues) {
+      const matching = fields?.find((f) => isConnectionTypeDataField(f) && f.envVar === key);
+      if (!matching) {
+        unmatched.push({
+          type: ConnectionTypeFieldType.ShortText,
```

Comment:
we _could_ be smart. If it contains `\n` make it long text, or if it is `true|false` make it a boolean. But otherwise I think this works.